RiotSwiftUI/Modules/Room/PollHistory/View/PollHistory.swift

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+struct PollHistory: View {
+    @Environment(\.theme) private var theme
+    
+    @ObservedObject var viewModel: PollHistoryViewModel.Context
+    
+    var bindings: PollHistoryViewBindings {
+        viewModel.viewState.bindings
+    }
+    
+    var body: some View {
+        VStack {
+            HStack {
+                SegmentedPicker(
+                    segments: PollHistoryMode.allCases.map { ($0.segmentTitle, $0) },
+                    selection: $viewModel.mode,
+                    interSegmentSpacing: 14
+                )
+                Spacer()
+            }
```

Comment:
You could remove the `HStack` and `Spacer` here instead applying the following frame modifier like you do on the views in the scroll view:
```
.frame(maxWidth: .infinity, alignment: .leading)
```

Reply:
Great catch! 
https://github.com/vector-im/element-ios/pull/7267/commits/da78ce6683b96d613a33f97f7d44110dddbfca23